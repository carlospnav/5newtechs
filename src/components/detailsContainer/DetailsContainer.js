import styled from "styled-components"
import { flexGroup } from "theme/utils"

export default styled.section`
  ${flexGroup("column")}
  color: white;
  margin-left: 20px;
  padding: 0 40px;
`