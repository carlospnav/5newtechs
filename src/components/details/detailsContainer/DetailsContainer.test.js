import { shallow } from "enzyme"
import DetailsContainer from "./DetailsContainer"
import "jest-styled-components"

describe("<DetailsContainer>", () => {
  it("should render a DetailsContainer properly.", () => {
    const wrapper = shallow(
      <DetailsContainer>
        <p>Containereeee!</p>
      </DetailsContainer>
    )
    expect(wrapper).toMatchSnapshot()
  })
})