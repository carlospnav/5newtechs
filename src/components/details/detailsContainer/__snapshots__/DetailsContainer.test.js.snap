// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailsContainer> should render a DetailsContainer properly. 1`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: null;
  -webkit-justify-content: null;
  -ms-flex-pack: null;
  justify-content: null;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: white;
  padding: 40px 60px;
  width: 100%;
}

@media (min-width:1024px) {
  .c0 {
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
    width: 50%;
    padding: 25px 10px;
  }
}

<section
  className="c0"
>
  <p>
    Containereeee!
  </p>
</section>
`;
